tmpl/wiki/weekly/2015-09-30.md

```diff
@@ -0,0 +1,186 @@
+30th September 2015: Functoria, Irmin plans, numbering and Pioneers
+
+### Agenda ###
+
+- Functoria
+- Irmin roadmap
+- Version numbering
+- Pioneer Projects
+
+Attendees:
+Reynir Björnsson, Daniel Bünzli, Amir Chaudhry (chair), Justin Cormack,
+Thomas Gazagnaire, David Kaloper, Thomas Leonard, Anil Madhavapeddy,
+Mindy Preston, Dave Scott, Magnus Skjegstad, Guybrush Threepwood and
+Jim Tittlser
+
+
+### Notes ###
+
+#### Functoria ####
+
+*See [mirage/mirage#441][]*
+
+There's been work towards a new EDSL that's captured in a library called
+Functoria.  We discussed this last time and it needed rebasing due to an
+earlier `.xl` patch being merged.
+
+Things have been cleaned up with the patch but it seems there have been some
+issues with Travis, which someone needs to take a closer look at.  Someone
+also needs to review and test the patch (Anil will do this, feedback from
+others also welcome). The examples should also be checked over.
+
+One thing we know is still missing as there's an opportunity to retain the
+shorthand flags we have for `--xen` and `--unix` as part of the configure step.
+ThomasG's not had a chance to look at that yet but would be great to keep them
+if we can.
```

Comment:
there's also some integration with environment variable that have been done by Daniel: https://github.com/dbuenzli/cmdliner/commit/e82fd02842719167c3a04fc7ed0edf9a8036203c